src/main/java/com/attacktimer/AttackTimerBarOverlay.java

```diff
@@ -86,9 +91,23 @@ public Dimension render(Graphics2D graphics)
             ratio = (float)Math.max(1.0f - ratio, 0f);
         }
 
+        if (client.getSpriteOverrides().containsKey(SpriteID.HEALTHBAR_DEFAULT_FRONT_30PX)) {
+            final int barWidth = HD_FRONT_BAR.getWidth();
+            final int barHeight = HD_FRONT_BAR.getHeight();
+            final int barX = canvasPoint.getX() - barWidth / 2;
+            final int barY = canvasPoint.getY() + 7;
+
+            // Include padding so the bar doesn't show empty at very low prayer values
```

Comment:
Is this comment copy/pasted from the HD prayer bar?

Reply:
Yes, my mistake i will remove this feature as it doesn't stay consistence with regular bar.

Reply:
![image](https://github.com/user-attachments/assets/97384acb-4f1a-443d-8c7a-d2adab0fdc5b)

Reply:
> Is this comment copy/pasted from the HD prayer bar?

like almost all the code of the current version 🤣 look at line 95 of AttackTimerBarOverlay